### ISSUE TITLE
MIDI

### ISSUE THREAD

**Issue by t0maszduda** [djuseeq/Ch376msc#25]
Hi 

Is it possible sent MIDI data from Ch376 via TX from Arduino to USB MIDI iNTERFACE?
![IMG_4458](https://user-images.githubusercontent.com/56415244/77749069-b15b3e00-7021-11ea-8cc3-0fd47c3a8611.jpeg)
 
Thanks
Tomasz

**Comment by t0maszduda** [comment]
as this:
http://www.hobbytronics.co.uk/usb-host-board-v24

**Comment by djuseeq** [comment]
Hi Tomasz,
sorry but with this library is not possible. This lib support just only storage controller mode.

**Comment by t0maszduda** [comment]
Yes I know. The question is is it possible at all  do that without checking firmware? 

Do you can make a little library for midi
which sends Serial.begin(31250) from Arduino by tx straight to usb socket

Without any command for read/write files etc.

**Comment by t0maszduda** [comment]
I found something but this is USB Host Shield and I don't know is it similar to ch376 or not
https://github.com/felis/USB_Host_Shield_2.0

**Comment by djuseeq** [comment]
Yes, it's possible to use with HID devices, i had an attempt in the past to write a code but i never finished. Takes too long time, and at the moment i don't have enough time. I would like to finish and if it succeeds I will make it public. With the shield in your link is possible also if you can find a lib, for file manage the ch376 is more easier because we don't need to bother with low level USB protocol or file system, just send the proper commands to the chip so our code takes up much less space in MCU.

**Comment by t0maszduda** [comment]
Thanks 

I’m waiting for MIDI library:)

Best Regards
Tom
